packages/cache/src/index.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 import { StyleSheet } from '@emotion/sheet'
-import { type EmotionCache, type SerializedStyles } from '@emotion/utils'
+import { type SerializedStyles } from '@emotion/serialize'
+import { type EmotionCache } from '@emotion/cache'
```

Comment:
For instance, this is complaining because `SerializedStyles` is not exported in `@emotion/serialize`, but that shouldn't be the case unless they are referencing the current version, but does this mean I need to bump them first or something? I literally have no idea.

Next is this `@emotion/cache`, since I'm inside the package already, I tried using `../types` instead but got an error. Where should I declare them? 🤔

Reply:
This can be disregard as this is a mistake on my part